restaurantSimulation/src/main/java/Generators/MenuGenerator.java

```diff
@@ -0,0 +1,41 @@
+package Generators;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import model.menu.JsonMenuHelper;
+import model.menu.Menu;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class MenuGenerator {
+	public MenuGenerator() throws IOException {
+	}
+	public static String readMenuJsonIntoString() throws IOException {
+		String json = Files.readString(new File("/Users/bcinternship2/git/Ostonya-Thomas/restaurantSimulation/src/main/resources/menu.json").toPath());
+		return json;
+	}
+
+
+	public static Menu generateMenu() throws IOException {
+		Gson gson = new GsonBuilder().setPrettyPrinting().create();
+		JsonMenuHelper helper = gson.fromJson(readMenuJsonIntoString(), JsonMenuHelper.class);
+		Menu menu = helper.convertToMenu();
+		return menu;
```

Comment:
Nitpick: you could do 
```suggestion
		return helper.convertToMenu();
```